examples/attention/opencog.conf

```diff
@@ -0,0 +1,137 @@
+#
+# This file provides an example configuration for a production OpenCog
+# server. Particularly noteworthy is the list of automatically loaded
+# modules.
+#
+SERVER_PORT           = 17001
+LOG_FILE              = /tmp/cogserver.log
+
```

Comment:
I'll move it to experiments/attention I just didn't want it to be in opencog/attention/experiments where it would have to be compiled every time when building opencog without needing it most of the time.